### DISCUSSION TITLE
Tags or categories for maps

### DISCUSSION THREAD

**Discussion by dtra82** [MKT-Playbook/Hub#22] [Feature requests]
I was thinking about this with the upcoming holiday tour, which could possibly be winter themed.
It’d be nice if tracks could have tags added like winter/ice/snow/food/summer/animal, etc, then items could be filtered on tags, could be useful for seasonal tours, and could also be used to apply to nitro tracks for filtering.
What do you think?

**Comment by TitusPosiadly** [comment]
Yeah, I like that idea. Although it would be another thing I would need to add to every track new and past which I can't spend time on right now. It would be another reason to finally open source the database, so everyone can edit it - kinda like a wiki. That way I could add a lot more data types like this one and would have more time to work on the actual app. It's just one hell of a task, but something I wanted to do for quite a while now. I will do some more research on that and will try to figure something out.
